src/db/mod.rs

```diff
@@ -1055,6 +1248,7 @@ impl DatabaseWrapper {
             }
 
             GenericDatabase::Fjall { keyspace, db } => {
+                // TODO: add option to write through transactions
```

Comment:
`db` is transactional, so the .insert() already creates a transaction internally